.hashibot.hcl

```diff
@@ -425,7 +425,7 @@ behavior "regexp_issue_labeler_v2" "service_labels" {
       "aws_route53_([^d]|d[^o]|do[^m]|dom[^a]|doma[^i]|domai[^n]|domain[^s]|domains[^_]|[^r]|r[^e]|re[^s]|res[^o]|reso[^l]|resol[^v]|resolv[^e]|resolve[^r]|resolver[^_])",
     ],
     "service/route53domains" = [
-      "aws_route53_domains_",
+      "aws_route53_domain",
```

Comment:
Our preference for naming these resources will be `aws_route53domains_THING`, e.g. `aws_route53domains_domain`

Reply:
For consistency with Route53 Resolver - wouldn't it then be better to go with `aws_route53_domains_domain`? (although that feels a bit repetitive too 😄)

Reply:
Going forward, we'd really like to get in the habit of `aws_SERVICE_THING` to more align with CloudFormation resources as well as prevent any oddities of V2 APIs. See also: https://github.com/terraform-providers/terraform-provider-aws/pull/8842#discussion_r390546009

Reply:
Updated to `aws_route53domains_` to match service name